packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.tsx

```diff
@@ -0,0 +1,187 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+/* tslint:enable:no-unused-variable */
+import * as PropTypes from 'prop-types';
+import {
+  autobind,
+  BaseComponent,
+  css,
+  findScrollableParent
+} from '../../Utilities';
+import { IScrollablePaneProps } from './ScrollablePane.Props';
+import { Sticky } from '../../Sticky';
+import * as stylesImport from './ScrollablePane.scss';
+const styles: any = stylesImport;
+
+export class ScrollablePane extends BaseComponent<IScrollablePaneProps, {}> {
+  public static childContextTypes = {
+    subscribe: PropTypes.func,
+    addStickyHeader: PropTypes.func,
+    removeStickyHeader: PropTypes.func,
+    addStickyFooter: PropTypes.func,
+    removeStickyFooter: PropTypes.func
+  };
+
+  public refs: {
+    root: HTMLElement;
+    stickyContainer: HTMLElement;
+    stickyAbove: HTMLElement;
+    stickyBelow: HTMLElement;
+  };
+
+  private _stickyContainer: HTMLElement;
+  private _scrollElement: HTMLElement;
+  private _subscribers: Function[];
+  private _topHeaderHeight: number;
+  private _bottomFooterHeight: number;
+  private _stickyAbove: Sticky[];
+  private _stickyBelow: Sticky[];
+
+  public getChildContext() {
+    return {
+      subscribe: this.subscribe,
+      addStickyHeader: this.addStickyHeader,
+      removeStickyHeader: this.removeStickyHeader,
+      addStickyFooter: this.addStickyFooter,
+      removeStickyFooter: this.removeStickyFooter,
+    };
+  }
+
+  constructor(props: IScrollablePaneProps) {
+    super(props);
+    this._subscribers = [];
+    this._topHeaderHeight = 0;
+    this._bottomFooterHeight = 0;
+    this._stickyAbove = [];
+    this._stickyBelow = [];
+  }
+
+  public componentDidMount() {
+    const { stickyAbove, stickyBelow, root, stickyContainer } = this.refs;
+
+    this._scrollElement = findScrollableParent(root);
+    if (this._scrollElement) {
+      this._stickyContainer = stickyContainer;
+      this._events.on(this._scrollElement, 'scroll', this._notifySubscribers);
+      this._events.on(window, 'resize', () => {
+        setTimeout(() => {
+          this._resizeContainer();
+          this._notifySubscribers();
+          this._setDistances(this._stickyAbove, stickyAbove);
+          this._setDistances(this._stickyBelow, stickyBelow);
+        }, 5);
+      });
+      setTimeout(() => {
+        this._resizeContainer();
+        this._stickyContainer.parentElement.removeChild(this._stickyContainer);
+        this._scrollElement.parentElement.insertBefore(this._stickyContainer, this._scrollElement.nextSibling);
+        this._notifySubscribers();
+        this._sortFooters();
+      }, 500);
+    } else {
+      throw new TypeError('Expected ScrollablePane to be within a parent element with data-is-scrollable=true');
```

Comment:
To me, `ScrollablePane` should supply the element with `data-is-scrollable="true"`, not depend on it. It should also supply the standard `overflow` and `-webkit-overflow-scroll: touch` behaviors.